playbooks/common/openshift-master/config.yml

```diff
@@ -5,6 +5,19 @@
     t_oo_option_master_debug_level: "{{ lookup('oo_option', 'openshift_master_debug_level') }}"
 
   pre_tasks:
+  # Per https://bugzilla.redhat.com/show_bug.cgi?id=1469336
+  #
+  # When scaling up a cluster upgraded from OCP <= 3.5, ensure that
+  # OPENSHIFT_DEFAULT_REGISTRY is present as defined on the existing
+  # masters, or absent if such is the case.
+  - name: Detect if this host is a new master in a scale up
+    set_fact:
+      g_openshift_master_is_scaleup: "{{ ( openshift.common.hostname in ( groups['new_masters'] | default([]) ) ) | bool }}"
```

Comment:
Probably should return bool by default. I was just kinda covering my own butt just in case. I can try it without all of the extra paren's and the extra `...) | bool`